.vib/postgrest/goss/postgrest.yaml

```diff
@@ -0,0 +1,9 @@
+command:
+  check-app-version:
+    exec: postgrest --help
+    exit-status: 0
+    stdout:
+      - {{ .Env.APP_VERSION }}
+user:
+  postgrest:
+    exists: true
```

Comment:
The logic for adding the user is found in the Dockerfile itself:

https://github.com/bitnami/containers/blob/60ad55114dc731bc6689c4e8e97864e1a420baa7/bitnami/postgrest/10/debian-11/Dockerfile#L39

I just added it because it caught my attention, but it is true that according to what we agreed on, testing commands directly found in the Dockerfile should be avoided.